emmet-core/emmet/core/openmm/tasks.py

```diff
@@ -0,0 +1,217 @@
+        None,
+        description=(
+            "The friction coefficient for the integrator " "(inverse picoseconds)."
+        ),
+    )
+
+    platform_name: Optional[str] = Field(
+        None,
+        description=(
+            "The name of the OpenMM platform to use, passed to "
+            "Interchange.to_openmm_simulation."
+        ),
+    )
+
+    platform_properties: Optional[dict] = Field(
+        None,
+        description=(
+            "Properties for the OpenMM platform, passed to "
+            "Interchange.to_openmm_simulation."
+        ),
+    )
+
+    state_interval: Optional[int] = Field(
+        None,
+        description=(
+            "The interval for saving simulation state. For no state, set to 0."
+        ),
+    )
+
+    state_file_name: Optional[str] = Field(
+        None, description="The name of the state file to save."
+    )
+
+    traj_interval: Optional[int] = Field(
+        None,
+        description=(
+            "The interval for saving trajectory frames. For no trajectory, set to 0."
+        ),
+    )
+
+    wrap_traj: Optional[bool] = Field(
+        None, description="Whether to wrap trajectory coordinates."
+    )
+
+    report_velocities: Optional[bool] = Field(
+        None, description="Whether to report velocities in the trajectory file."
+    )
+
+    traj_file_name: Optional[str] = Field(
+        None, description="The name of the trajectory file to save."
+    )
+
+    traj_file_type: Optional[str] = Field(
+        None,
+        description="The type of trajectory file to save. Options are 'dcd' and 'h5'.",
+    )
+
+
+class CalculationOutput(BaseModel):
+    """OpenMM calculation output files and extracted data."""
+
+    dir_name: Optional[str] = Field(
+        None, description="The directory for this OpenMM task"
+    )
+
+    dcd_file: Optional[str] = Field(
+        None, description="Path to the DCD file relative to `dir_name`"
+    )
+
+    state_file: Optional[str] = Field(
+        None, description="Path to the state file relative to `dir_name`"
+    )
+
+    steps: Optional[list[int]] = Field(
+        None, description="Steps where outputs are reported"
+    )
+
+    time: Optional[list[float]] = Field(None, description="List of times")
+
+    potential_energy: Optional[list[float]] = Field(
+        None, description="List of potential energies"
+    )
+
+    kinetic_energy: Optional[list[float]] = Field(
+        None, description="List of kinetic energies"
+    )
+
+    total_energy: Optional[list[float]] = Field(
+        None, description="List of total energies"
+    )
+
+    temperature: Optional[list[float]] = Field(None, description="List of temperatures")
+
+    volume: Optional[list[float]] = Field(None, description="List of volumes")
+
+    density: Optional[list[float]] = Field(None, description="List of densities")
+
+    elapsed_time: Optional[float] = Field(
+        None, description="Elapsed time for the calculation"
+    )
```

Comment:
Is it possible to specify units here? Or does it depend on how the calc input is set up?